cirq/contrib/acquaintance/executor_test.py

```diff
@@ -23,21 +23,22 @@
 import cirq.contrib.acquaintance as cca
 
 
-class ExampleGate(cirq.Gate):
+class ExampleGate(cirq.MultiQubitGate):
     def __init__(self, wire_symbols: Sequence[str]) -> None:
+        super(ExampleGate, self).__init__(len(wire_symbols))
```

Comment:
just `super()`

Reply:
Ah python 3.  Fixed here and throughout.